docs/weave-tutorial/02-domain.md

```diff
@@ -6,22 +6,24 @@ sidebar_label: Domain
 
 The first thing we consider is the data we want to store, which is known as the state. After that, we can focus on the messages, which trigger state transitions. All blockchain state must be stored in our Merkle-ized store, which provides validity hashes and proofs. This is exposed to the application as a basic key-value store, which also allows in-order iteration over the keys. On top of this, we have built some tools, such as secondary indexes and sequences, similar to how [Storm adds an ORM](https://github.com/asdine/storm#simple-crud-system) on top of [BoltDB’s key-value store](https://github.com/boltdb/bolt#using-buckets). We have avoided struct tags and tried to type as strictly as we can (without using generics).
 
-We are building an Order book application. This topic is quite hot at the time of writing this tutorial.
+## State
 
-- **Markets** will contain rules as to who (which public keys) may create an **order book** or who can cancel an order. The blockchain may have multiple markets, and each market may have multiple Orderbooks; however, each token pair can only have one order book per market.
+State as in [Turing machine](https://en.wikipedia.org/wiki/Turing_machine), is how the system's data is presented this contained. We are building an Order book application to solidify the state concept. Exchange topic is quite hot at the time of writing this tutorial.
```

Comment:
I don't understand hw this first sentence ends: 'this contained' 

Looks like a typo?